uint32 index
---
MapArea area
geometry_msgs/Polygon[] prohibited_areas
